code/src/components/Counter.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { CounterContainer, CounterText } from './StyledComponents/CounterStyling';
+
+export const Counter = () => {  
+  const todos = useSelector(state => state.todos.items)
+  const leftTodo = todos.filter(todo => todo.complete === false)
+
+  return(
+    <CounterContainer> 
+      <CounterText> 
+        You have {leftTodo.length} remaining tasks out of {todos.length} to do!
+      </CounterText>
+    </CounterContainer>
+  );
+};
```

Comment:
great that you have this code in its own components file! 
+ great indentation, really clean!